test/recipes/70-test_sslmessages.t

```diff
@@ -28,6 +28,10 @@ plan skip_all => "$test_name needs the sock feature enabled"
 plan skip_all => "$test_name needs TLS enabled"
     if alldisabled(available_protocols("tls"));
 
+#We assume the EC extensions are sent
+plan skip_all => "$test_name needs EC enabled"
+    if disabled("ec");
+
```

Comment:
Isn't `70-test_sslmessages` a general test of tls messages?  In that case, I think this is a bad idea.  There are other options where this particular test fails, shall we skip this test for them as well?  That would be counter productive...